src/Microsoft.ML.StandardLearners/Standard/Online/AveragedPerceptron.cs

```diff
@@ -24,12 +24,12 @@
 
 namespace Microsoft.ML.Trainers.Online
 {
-    // This is an averaged perceptron classifier.
-    // Configurable subcomponents:
-    //     - Loss function. By default, hinge loss (aka max-margin avgd perceptron)
-    //     - Feature normalization. By default, rescaling between min and max values for every feature
-    //     - Prediction calibration to produce probabilities. Off by default, if on, uses exponential (aka Platt) calibration.
-    /// <include file='doc.xml' path='doc/members/member[@name="AP"]/*' />
+    /// <summary>
+    /// This is averaged perceptron trainer.
```

Comment:
>This is averaged perceptron trainer. [](start = 8, length = 36)

See the email on standard summary text.

Reply:
+1

---
In reply to: [256223833](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256223833) [](ancestors = 256223833)

Reply:
updated

---
In reply to: [256251639](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256251639) [](ancestors = 256251639,256223833)